finan/models/finan_retorno.py

```diff
@@ -705,6 +705,25 @@ def processar_retorno(self):
         # Processa os boletos do arquivo
         for boleto in arquivo_retorno.boletos:
 
+            # verifica se o boleto é duplicado
+            if boleto.pagamento_duplicado == True:
+                boleto.pagamento_duplicado = 'SIM'
+            elif boleto.pagamento_duplicado == False:
+                boleto.pagamento_duplicado = 'NÃO'
+
+            # verifica exixtencia de data de crédito e inverte se existir
+            if boleto.data_credito is not None:
+                boleto.data_credito_fmt = formata_data(boleto.data_credito)
+            elif boleto.data_credito is None:
+                boleto.data_credito_fmt = ''
+
+            # verifica exixtencia de data de ocorrencia e inverte se existir
+            if boleto.data_ocorrencia is not None:
```

Comment:
Em vez de " is not None", use " if boleto.data_ocorrencia: " direto. No elif, use o " not " na frente: "if not boleto.data_ocorrencia: ".